src/main/java/org/umssdiplo/automationv01/core/managepage/Accident/CreateAccident.java

```diff
@@ -0,0 +1,91 @@
+package org.umssdiplo.automationv01.core.managepage.Accident;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.umssdiplo.automationv01.core.managepage.BasePage;
+import org.umssdiplo.automationv01.core.utils.CommonEvents;
+
+public class CreateAccident extends BasePage {
+
+    @FindBy(id = "employeeDropDown")
+    private WebElement employeeDropDown;
+
+    @FindBy(xpath = "//*[@id=\"mat-option-10\"]/span")
+    private WebElement employee;
+
+    @FindBy(id = "departmentDropDown")
+    private WebElement departmentDropDown;
+
+    @FindBy(xpath = "//*[@id=\"mat-option-0\"]/span")
```

Comment:
1. Dont use '*', you should Select tag on your xpath. Example div, input, ul,li, etc.
2. User simple quote instead of double quotes
3. in your xpath is using ID, this means that you can use ID directly